temp/static_analysis.py

```diff
@@ -0,0 +1,340 @@
+"""Pseudocode for PyDPF-Post Static Mechanical Simulations APIs."""
+
+import ansys.dpf.post as dpf
+from ansys.dpf.post import examples
+from ansys.dpf.post.selection import Selection
+
+# Provide Public Enums for available physics_types and analysis_types
+# from ansys.dpf.post import physics_types, analysis_types
+# instead of having them private
+# from ansys.dpf.post.common import _AnalysisType, _PhysicsType
+
+# Load the simulation files
+static_simulation = dpf.load_simulation(examples.simple_bar)
+# static_simulation = dpf.load_simulation(examples.simple_bar,
+#                                         physics_type=physics_types.mechanical,
+#                                         analysis_type=analysis_types.static)
+
+# -----------------------------------------------------------------------------------------
+# Explore the simulation metadata
+
+# Print information about the simulation
+print(static_simulation)
+
+# Print the mesh
+print(static_simulation.mesh)
+
+# Print the list of constructed geometries
+print(static_simulation.geometries)
+
+# Print the list of boundary conditions
+print(static_simulation.boundary_conditions)
+
+# Print the list of loads
+print(static_simulation.loads)
+
+# Print the list of available named selections
+named_selections = static_simulation.named_selections
+print(named_selections)
+
+# Print the list of steps (the time_freq_support)? the list of sub-steps? both? the times?
+print(static_simulation.time_freq_support)
+print(static_simulation.steps)
+
+# Print the list of available results
+print(static_simulation.results)
+
+# General plot of the simulation object with, by default:
+# - the mesh, not deformed, at step 0
+# - the geometry
+# - the boundary conditions
+# static_simulation.plot(mesh=True, geometry=True, boundary_conditions=True)
+
+
+# -----------------------------------------------------------------------------------------
+# Create and use geometry
+# Creation of geometry is for now in PyDPF-Core.
+# Do we expose those in Post in some way? Simply by importing them in a specific Post module?
+from ansys.dpf.core.geometry import Line
+
+line = Line(coordinates=[[1.0, 0.0, 1.0], [5.0, 0.0, 1.0]])
+static_simulation.geometries.append(line)
+# static_simulation.plot(mesh=True, geometry=True, boundary_conditions=True)
+
+
```

Comment:
I'm not sure this is needed/supported at this point.

Adding a geometry might seem we can change the mesh with the new geometry and solve?

I rather keep the geometry as "only read" for the moment.

Reply:
The goal of the constructed geometry is to allow for creation of supports for operations such as mapping and result evaluation on sections... etc.
It is part of what we will need the user to be able to add and interact with. Right now it is just a list of "geometry" objects, but it could be more.
To me, `geometry`, is different from `mesh` and as long as we document it properly it should be ok.
Also we do not have the possibility to generate input files from DPF.

But this definitely needs to be better defined in terms of what is needed.